indra/sources/tees/tees_api.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+"""
+This module provides a simplified API for invoking the Turku Event Extraction
+System (TEES) on text and extracting INDRA statement from TEES output.
+
+See publication:
+Jari Björne, Sofie Van Landeghem, Sampo Pyysalo, Tomoko Ohta, Filip Ginter, Yves Van de Peer, Sofia Ananiadou and Tapio Salakoski, PubMed-Scale Event Extraction for Post-Translational Modifications, Epigenetics and Protein Structural Relations. Proceedings of BioNLP 2012, pages 82-90, 2012.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+from builtins import dict, str
+import os
+from indra.sources.tees.processor import TEESProcessor
+
+import os.path
+import logging
+import codecs
+
+from indra.sources.tees.parse_tees import tees_parse_networkx_to_dot
+import networkx.algorithms.dag as dag
+
+logger = logging.getLogger('tees')
+
+# If TEES isn't specified, we will check to see if any of these directories
+# contain all of the files in tees_installation_files; if so, we'll assume
+# that it is a TEES installation.
+tees_candidate_paths = ['../TEES', '~/TEES', '~/Downloads/TEES']
+tees_installation_files = ['batch.py', 'classify.py', 'train.py',
+        'visualize.py']
+tees_installation_dirs = ['Classifiers', 'Detectors', 'Evaluators', 'Core']
+
+def tees_process_text(text, pmid=None, tees_path=None, python2_path=None):
```

Comment:
We can rename this to just `process_text`, the module path makes it clear that it's done by tees